tests/test_odd_even.py

```diff
@@ -17,7 +19,6 @@ def test_odd_even():
 
     assert_allclose(odd[0], 0.5)
     assert_allclose(even[0], 0.5)
-    assert np.isnan(sigma)
```

Comment:
Why is this check removed?

Reply:
Because the test was wrong.  sigma should be zero, not nan. I'll put in a new test.